File: Benchmarks/enterprise/java/jakarta/testcode/BenchmarkTest52969.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import jakarta.ws.rs.*;
import jakarta.ws.rs.core.Context;
import jakarta.ws.rs.core.MediaType;
import jakarta.ws.rs.core.Response;

@Path("/")
public class BenchmarkTest52969 {

    private static String stripWhitespace(String v) { return v.strip(); }

    @POST
    @Path("/BenchmarkTest52969")
    @Consumes(MediaType.APPLICATION_XML)
    @Produces(MediaType.APPLICATION_JSON)
    public Response BenchmarkTest52969(String xmlBody, @Context HttpServletRequest request, @Context HttpServletResponse response) throws Exception {
        String xmlValue = xmlBody;
        String data = stripWhitespace(xmlValue);
        response.setContentType("text/html");
        return Response.ok(data, MediaType.TEXT_HTML).build();
    }
}

File: Benchmarks/enterprise/python/django/testcode/benchmark_test_52969.py
# SPDX-License-Identifier: Apache-2.0
from django.http import JsonResponse


def BenchmarkTest52969(request, path_param):
    path_value = path_param
    data = (lambda v: v.strip())(path_value)
    result = 100 / int(str(data))
    return JsonResponse({"saved": True})
